packages/devtools_app/lib/src/screens/debugger/program_explorer_controller.dart

```diff
@@ -115,26 +115,34 @@ class ProgramExplorerController extends DisposableController
       if (result != null) {
         selectNode(result);
         result.expandAscending();
-        _selectedNodeIndex.value = _calculateNodeIndex(searchCondition);
+        _selectedNodeIndex.value = _calculateNodeIndex(
+          matchingNodeCondition: searchCondition,
+          includeCollapsedNodes: false,
+        );
         return;
       }
     }
   }
 
-  int _calculateNodeIndex(
-      bool matchingNodeCondition(VMServiceObjectNode node)) {
+  int _calculateNodeIndex({
+    bool matchingNodeCondition(VMServiceObjectNode node),
+    bool includeCollapsedNodes,
```

Comment:
give this a default value. otherwise, the ternary check `includeCollapsedNodes ? ... : ...` will fail on a null value for `includeCollapsedNodes`